app/views/report_templates/generate.html.erb

```diff
@@ -26,7 +26,16 @@
       label_help: _('Valid e-mail addresses delimited by "%s"') % ReportComposer::MailToValidator::MAIL_DELIMITER %>
   </div>
 
-  <%= select_f f, :format, ReportTemplateFormat.selectable, :id, :human_name, {}, :label => _('Output format'), :label_help => _('If the template supports format selection, user can choose preferred format.<br> Typically the template needs to use report_render macro.<br><br>If usage of this macro is not found in the template,<br>this field is disabled and the output format defaults to plain text.<br><br>If the template supports filter selection, but does not use the report_render macro,<br>in order to enable this field, add a comment to the template mentioning report_render macro name.'), :disabled => !@composer.template.supports_format_selection? %>
+  <% render_present = @composer.template.supports_format_selection?
+     output_formats = render_present ? ReportTemplateFormat.selectable : []
+     label_help = 'If the template supports format selection, user can choose preferred format.<br>
+                   Typically the template needs to use report_render macro.<br><br>
+                   If usage of this macro is not found in the template,<br>
+                   this field is disabled and the output format defaults to plain text.<br><br>
+                   If the template supports filter selection, but does not use the report_render macro,<br>
+                   in order to enable this field, add a comment to the template mentioning report_render macro name.'
```

Comment:
This string needs to be extracted for translation, see https://projects.theforeman.org/projects/foreman/wiki/Translating#Translating-for-developers for details

Reply:
Yeah, I believe @stejskalleos meant to keep extraction in select_f below, I see `_(label_help)` but that wouldn't work. We need to move the extraction here too.

Reply:
Yeah, Marek is right, didn't know that `_(variable)` wouldn't work, label reverted back as it was before.